1417/bn.md

```diff
@@ -0,0 +1,89 @@
+# LOJ 1417 - Forwarding Emails
+
+প্রথমেই ধরে নেই তোমার কাছে কিছু মানুষ আছে এবং তাদের পাঠানোর জন্য কিছু মেইল দেওয়া আছে।
+এখন মনেকরি, তুমি জানো তুমি প্রথম জনকে মেইল পাঠালে সে ২য় জনকে মেইল টা ফরোয়ার্ড করতে পারবে।
+এখানে একজন আরেকজনকে মেইল পাঠাতে পারবে তখনই যখন তাদের মধ্যে একটা যোগাযোগ আছে।
+ধরি, ১ম জন আর ২য় জনের মধ্যে যোগাযোগ আছে এবং ২য় আর ৩য় জনের মধ্যে যোগাযোগ আছে কিন্তু ১ম আর ৩য় জনের মধ্যে কোনো যোগাযোগ নেই। 
+তাহলে ১ম জন ৩য় জনকে মেইল পাঠাতে গেলে ২য় জনের মাধ্যমে পাঠানো লাগবে।
+এখন এই প্রবলেমে তোমাকে বলে দেওয়া থাকবে, কার সাথে কার যোগাযোগ আছে অর্থাৎ কে কাকে মেইল পাঠাতে পারবে।
+তোমাকে বলতে হবে কাকে প্রথমে মেইল পাঠালে তুমি বেশি সংখ্যক মানুষের কাছে মেইল পৌঁছাতে পারবা?
+আমরা একটা ছোট্ট উদাহরণ এর মাধ্যমে জিনিসটা বোঝার চেষ্টা করি। 
+
+উদাহরণঃ 
+১ ২ - ১ম জন ২য় জনকে মেইল পাঠাতে পারবে 
+২ ১ - ২য় জন ১ম জনকে মেইল পাঠাতে পারবে
+৫ ৩ - ৫ম জন ৩য় জনকে মেইল পাঠাতে পারবে
+৩ ৪ - ৩য় জন ৪র্থ জনকে মেইল পাঠাতে পারবে
+৪ ৫ - ৪র্থ জন ৫ম জনকে মেইল পাঠাতে পারবে
```

Comment:
* Try including a drawing of the graph. https://csacademy.com/app/graph_editor/
* Markdown makes these 5 lines into 1, resulting in poor readability. Use `<br/>` I guess.